lib/chatops/chatops.go

```diff
@@ -45,6 +45,11 @@ func (c *ChatOps) PostMessage(code int, target string) (*ChatOpsResponse, error)
 		}
 		resp.Channel = res.Channel
 		resp.Timestamp = strconv.FormatInt(res.Ts, 10)
+	case "teams":
+		_, err := c.teams(code, target, output)
```

Comment:
What is this unused value that is being returned and not set to a variable? Why would it not be needed?